supportedBackends/google/pipelines/v2alpha1/src/main/scala/cromwell/backend/google/pipelines/v2alpha1/GenomicsFactory.scala

```diff
@@ -260,12 +263,12 @@ object GenomicsFactory {
     *
     * When updating this value, also consider updating the CromwellImagesSizeRoundedUpInGB below.
     */
-  val CloudSdkImage = "gcr.io/google.com/cloudsdktool/cloud-sdk:276.0.0-slim"
+  val CloudSdkImage: String = if (config.hasPath("cloud-sdk-image-url")) { config.getString("cloud-sdk-image-url").toString } else "gcr.io/google.com/cloudsdktool/cloud-sdk:276.0.0-slim"
 
   /*
    * At the moment, the cloud-sdk:slim (727MB on 2019-09-26) and possibly stedolan/jq (182MB) decompressed ~= 1 GB
    */
-  val CromwellImagesSizeRoundedUpInGB = 1
+  val CromwellImagesSizeRoundedUpInGB: Int = if (config.hasPath("cloud-sdk-image-size-gb")) { config.getInt("cloud-sdk-image-size-gb") } else 1
```

Comment:
I suspect you can leave this at 1 unless you're hosting a vastly different sized image

Reply:
Yes we could leave it to 1, but possibility to update it in config doesn't hurt. Personally we don't need that option, but I prefer not to leave hardcoded value based original use case and force someone to do new pull request to fix just that. After all, it has default value -- no need to change it in most cases.

Reply:
Sure, your choice! 😄